plugins/pulsar-records-storage/src/test/java/com/github/bsideup/liiklus/pulsar/PulsarRecordsStorageTest.java

```diff
@@ -110,4 +127,36 @@ void shouldPreferEventTimeOverPublishTime() throws Exception {
             assertThat(it.getTimestamp()).isEqualTo(eventTimestamp);
         });
     }
+
+
+    @Target({ElementType.TYPE, ElementType.METHOD})
+    @Retention(RetentionPolicy.RUNTIME)
+    @ExtendWith(DisabledUntil.DisabledCondition.class)
+    public @interface DisabledUntil {
+        String value();
+
+        String comment();
+
+        class DisabledCondition implements ExecutionCondition {
+
+            @Override
+            public ConditionEvaluationResult evaluateExecutionCondition(ExtensionContext context) {
+                Optional<DisabledUntil> until = findAnnotation(context.getElement(), DisabledUntil.class);
+                if (until.isPresent()
```

Comment:
`until.isPresent()` is not needed, `.map` will evaluate only if present

Can be as simple as:
```
if (until.map(it -> LocalDate.parse(it.value()).isAfter(LocalDate.now())).orElse(false)) {
```

Reply:
lol, definitely